doc/arch/adr-001-data-models.md

```diff
@@ -0,0 +1,101 @@
+# IOW Entity Relationship Diagram
+
+## Context
+
+Below is the Entity Relationship Diagram illustrating the initial data model as discussed in Issue #69. Not included in the diagram at the time of creation are the file fields for `ReferenceImage` and `Submission` that would reference those respective file S3 uploads (to be implemented in Issue #81).
+
+```mermaid
+erDiagram
+    Role {
+        string description
+    }
+    User ||--o{ User_Utility : "has"
+    User }o--|{ State : "within"
+    User }o--|| Role : is
+    User {
+        Role role FK
+        email email
+        boolean is_staff
+        boolean is_active
+        date date_joined
+        password password
+        boolean is_locked_out
+        phone phone
+        State state FK
+        Utility utilities
+    }
+    Utility ||--o{ User_Utility : "belongs to"
+    Utility }o--|| State : "within"
+    Utility {
+        string pwsid
+        string name
+        address address
+        point location
+        State state FK
+    }
+    User_Utility {
```

Comment:
Since `User_Utility` doesn't have any additional columns beyond each's id, we can drop it from here, and just indicate a many-to-many relationship between User and Utility